databases/week2/library-query.sql

```diff
@@ -0,0 +1,147 @@
+use library;
+SELECT
+  *
+FROM
+  author;
+-- insert author
+INSERT INTO
+  author (name, email)
+VALUES
+  (
+    "Aarika Ellingworth",
+    "aellingworth0@harvard.edu"
+  );
+INSERT INTO
+  author (name, email)
+VALUES
+  ('Pren Goldsworthy', 'pgoldsworthy1@spotify.com'),
+  ('Pablo Kisbee', 'pkisbee2@lulu.com'),
+  ('Rodie Duncan', 'rduncan3@quantcast.com');
+-- genre
+INSERT INTO
+  genre(name)
+VALUES
+  ("Action and Adventure"),("languages"),("Classics"),("Comic Book or Graphic Novel");
+-- member
+INSERT INTO
+  member(name, phone, email)
+VALUES
+  ("sunny", "55668963", "sunny@163.com"),("ammy", "23368963", "sunny@163.com"),("jone", "48958963", "jone@163.com"),("lina", "45268963", "lina@163.com"),("fie", "25768963", "fie@163.com");
+-- rack
+INSERT INTO
+  rack(categary)
+VALUES
+  ("a-d"),("e-i"),("j-n"),("o-t"),("u-z");
+-- book
+ALTER TABLE
+  book
+MODIFY
+  COLUMN price INT NOT NULL;
+UPDATE
+  book
+SET
+  price = 199
+WHERE
+  id = 9;
+-- change the price to varchar type
```

Comment:
as you commented, if you want to add the currency, you have to change it into varchar or use money datatype, however as the MySQL doesnt support Money, you may change it to Decimal, Flout or Double and ass a Column, call Currency